### DISCUSSION TITLE
InfiniteChestsV3 does not work with Tshock 4.5.2

### DISCUSSION THREAD

**Discussion by wwa171** [Pryaxis/TShock#2295] [Q&A] [answered]
`System.MissingMethodException: 메서드를 찾을 수 없습니다. 'TShockAPI.ConfigFile TShockAPI.TShock.get_Config()' 위치: InfiniteChestsV3.DB.Connect() 위치: InfiniteChestsV3.InfMain.OnGameInitialize(EventArgs args) 위치: TerrariaApi.Server.HandlerCollection1.Invoke(ArgsType args) 파일 D:\a\TShock\TShock\TerrariaServerAPI\TerrariaServerAPI\TerrariaApi.Server\HandlerCollection.cs:줄 109
`

Tshock turns off immediately when an error pops up. Is there any solution?

**Comment by Terrabade** [comment]
The latest version of TShock changed the config structure, plugin’s outdated and the reference to it will need to be updated I believe either by the plugin creator or yourself.

**Comment by moisterrific** [comment, marked as answer]
https://github.com/moisterrific/InfiniteChestsV3/releases/tag/1.2.1
